wiki/es/es-Software-Manual.md

```diff
@@ -4,18 +4,15 @@ layout: wiki
 permalink: /wiki/Software-Manual
 title: 'Manual del Software'
 ---
-# Manual de Usuario de Jamulus
-
-{:.no_toc}
+# Manual de Usuario de Jamulus {:.no_toc}
```

Comment:
I hope we don't need to fix this manually every time.

"toc" seems to be "Table of contents"

Reply:
I formatted the version in EN so the processed version with po4a comes out correctly. It shouldn't need editing again.

Reply:
At least for ES it produces a wrong html file: 

![toc](https://user-images.githubusercontent.com/20726856/120239841-7a08a280-c25f-11eb-9dba-6004a17ca97e.png)

Reply:
Fixed it - hopefully. The EN version wasn't formatted properly. It's working ok now locally. This PR still needs some work to fix the mess in German though. I hope to get to it tomorrow.